apps/network-alerts/js/attention/attention.js

```diff
@@ -31,26 +30,24 @@ function sendNotification() {
   }
 
   return new Promise(function(resolve, reject) {
-    var title = params.title;
-    var body = params.body;
+    var titleL10n = params.title;
+    var bodyL10n = { raw: params.body };
     var req = navigator.mozApps.getSelf();
 
     req.onsuccess = function onsuccess(event) {
       var app = event.target.result;
 
-      var notification = new Notification(
-        navigator.mozL10n.get(title), {
-          body: body,
-          tag: '' + Date.now(), // needs to be unique
-          icon: NotificationHelper.getIconURI(app) + '?titleID=' + title
-        }
-      );
-
-      notification.onerror = function onerror() {
-        reject(new Error('CMAS: notification API error'));
-      };
-
-      notification.onshow = resolve;
+      NotificationHelper.send(titleL10n, { 
+        bodyL10n: bodyL10n,
+        tag: '' + Date.now(), // needs to be unique
+        icon: NotificationHelper.getIconURI(app) + '?titleID=' + titleL10n
```

Comment:
IIRC the NotificationHelper will close the notification after click automatically, unless we add `closeOnClick: false` for keeping it in the tray. Since the only why to dismiss the alert notification is using swipe gesture, please add `closeOnClick: false` to keep the behavior.